docs/index.rst

```diff
@@ -59,11 +59,19 @@ That done, you can run tests using ``make``::
     $ make test-foreman-ui
     $ make test-foreman-smoke
 
-If you want to run the test suite without the aid of ``make``, you can do that
-with either `unittest`_ or `nose`_::
+Robottelo provides two test suites. One for testing Robottelo itself and other
+for testing Foreman/Satellite 6. Robottelo's tests are under
+``tests/robottelo`` directory and Foreman/Satellite 6 tests are under
+``tests/foreman`` directory.
```

Comment:
Robottelo provides two test suites<ins>, one</ins> for testing Robottelo itself and <ins>an</ins>other for testing Foreman/Satellite 6. Robottelo's tests are under <ins>the</ins> ``tests/robottelo`` directory and <ins>the</ins> Foreman/Satellite 6 tests are under <ins>the</ins> ``tests/foreman`` directory.